frappe/public/js/frappe/form/grid_row.js

```diff
@@ -250,10 +251,247 @@ export default class GridRow {
 		}
 	}
 
+	add_column_configure_button() {
+		if (this.configure_columns) {
+			this.configure_columns_button = $(`
+				<div class="col grid-static-col col-xs-1 d-flex justify-content-center">
+					<a>${frappe.utils.icon('setting-gear', 'sm', '', 'filter: opacity(0.5)')}</a>
```

Comment:
use the "spanner" instead of gear?

Reply:
Gear looks better than spanner

### With gear

<img width="1046" alt="Screenshot 2021-09-08 at 12 50 39 PM" src="https://user-images.githubusercontent.com/8780500/132465528-b191251d-4f08-4edd-88d3-7e614700c19e.png">

### With spanner

<img width="1029" alt="Screenshot 2021-09-08 at 12 58 11 PM" src="https://user-images.githubusercontent.com/8780500/132465572-13b8d80b-b21d-4e7f-a169-0406d88780e0.png">